lib/Client/Auth.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace LightStepBase\Client;
+
+require_once(dirname(__FILE__) . "/Util.php");
+require_once(dirname(__FILE__) . "/../../thrift/CroutonThrift/Types.php");
+
+class Auth
```

Comment:
A "nice to have" would be to add a doc comment to these new classes. I know the old ones don't have it but they should. https://docs.phpdoc.org/references/phpdoc/basic-syntax.html

Reply:
Your PR with the proto files also can serve as an example

Reply:
done